instrumentation/github.com/aws/aws-sdk-go/service/config/config.go

```diff
@@ -0,0 +1,75 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
This seems like it does not need to be an independent package. Can this live in the `otels3` package?